orcid_hub/templates/CV.html

```diff
@@ -434,14 +434,39 @@
 		<td width="596" valign="top" style="border: 1px solid #000001; padding: 0cm 0.19cm">
       <ul>
         {% for j in work_type_journal %}
-        <li><font face="Arial, serif"><font style="font-size: 12pt">{{ j.get('publication-date', 'year', 'value')}}, {{
-          j.get('title', 'title', 'value') }}, <font style="font-size: 11pt"><i>{{ j.get('type') }}</i>
-            {% if j.get('external-ids') and  j.get('external-ids', 'external-id') %}
-              {% for e in j.get('external-ids', 'external-id') %}
+        <li><font face="Arial, serif"><font style="font-size: 12pt">
```

Comment:
"copy-paste" antipattern. This is repeated 5 more times! It needs to be refactored into a jinja macro.